node/pkg/chain/websocketchainreader/websocketreader.go

```diff
@@ -172,7 +180,7 @@ func processLogs(ctx context.Context, sub klaytn.Subscription, logs <-chan types
 	for {
 		select {
 		case err := <-sub.Err():
-			log.Error().Err(err).Msg("Error in subscription")
+			log.Warn().Err(err).Msg("Error in subscription")
```

Comment:
there could be different reasons, but think it is quite unhelpful error log since it can't be told from which endpoint this error is happening, and also it'll auto reconnect on this failure. Ending up thinking that this log doesn't deserve error level